sourmash_lib/test__minhash.py

```diff
@@ -176,14 +177,27 @@ def test_basic_dna_bad_2(track_abundance):
 
 def test_basic_dna_bad_force(track_abundance):
     # test behavior on bad DNA
-    mh = MinHash(1, 4, track_abundance=track_abundance)
+    mh = MinHash(100, 4, track_abundance=track_abundance)
```

Comment:
Added comment in https://github.com/swamidass/sourmash/pull/2 - we want to store multiple hashes.